src/Core/src/Platform/iOS/ButtonExtensions.cs

```diff
@@ -9,8 +9,7 @@ public static class ButtonExtensions
 
 		public static void UpdateStrokeColor(this UIButton platformButton, IButtonStroke buttonStroke)
 		{
-			if (buttonStroke.StrokeColor is not null)
-				platformButton.Layer.BorderColor = buttonStroke.StrokeColor.ToCGColor();
+			platformButton.Layer.BorderColor = buttonStroke?.StrokeColor?.ToCGColor() ?? UIColor.Clear.CGColor;
```

Comment:
While the same extension is used by the ImageButton, could you include in the sample (and test) an ImageButton too?

Reply:
Done, I've modified the UiTest

Reply:
Thanks. Triggered a new build.